Source/SwiftLintFramework/Rules/Lint/WeakDelegateRule.swift

```diff
@@ -81,7 +82,7 @@ public struct WeakDelegateRule: ASTRule, SubstitutionCorrectableASTRule, Configu
                 let offset = attribute.offset,
                 let length = attribute.length,
                 let value = file.stringView.substringWithByteRange(ByteRange(location: offset, length: length)),
-                value.hasPrefix("@UIApplicationDelegateAdaptor") {
+                (value.hasPrefix("@UIApplicationDelegateAdaptor") || value.hasPrefix("@NSApplicationDelegateAdaptor")) {
```

Comment:
can you please update this to handle `WKExtensionDelegateAdaptor` and fix https://github.com/realm/SwiftLint/issues/3611 as well? 🙏

Reply:
Done!